spec/javascripts/reducers/stories_spec.js

```diff
@@ -0,0 +1,51 @@
+import reducer from 'reducers/stories';
+import { toggleStory } from 'actions/story';
+
+const storiesArray = [
+  {
+    id: 1,
+    collapsed: true
+  },
+  {
+    id: 2,
+    collapsed: true
+  },
+  {
+    id: 3,
+    collapsed: true
+  }
+];
+
+describe('Stories reducer', () => {
+  describe("Toggle story", () => {
+    describe("When story is collapsed", () => {
+      it("expand story", () => {
+        const initialState = storiesArray;
+        const story = storiesArray[0];
+        story.collapsed = true;
```

Comment:
This is gonna mutate the story from the props, making a test share state with the others. You could do something like this:

```js
describe('Stories reducer', () => {
  let storiesArray;

  beforeEach(() => {
    storiesArray = [
      {
        id: 1,
        collapsed: true
      },
      {
        id: 2,
        collapsed: true
      },
      {
        id: 3,
        collapsed: true
      }
    ];
  });
  describe("Toggle story", () => {
    describe("When story is collapsed", () => {
      it("expand story", () => {
```
And then on every test it's gonna be a new array so no risk on mutating a story from the array 😄